extensions/wakatime/package.json

```
@@ -31,6 +31,14 @@
       "title": "Show Private Leaderboards",
       "description": "Show your private leaderboards",
       "mode": "view"
+    },
+    {
+      "name": "today-summary",
+      "subtitle": "WakaTime",
+      "title": "Show Today Summary",
+      "description": "Show your WakaTime ToDay Summary in Menu Bar",
```

```suggestion
      "description": "Show Today's Summary in the Menu Bar",
```
How about something like this as the description?